dockers/docker-snmp-sv2/Dockerfile.j2

```diff
@@ -18,7 +18,9 @@ ENV DEBIAN_FRONTEND=noninteractive
 # The file referenced (`/usr/share/dpkg/no-pie-compile.specs`) is in the `libdpkg-perl` package on Debian
 RUN apt-get update   && \
     apt-get install -y  \
-        curl            \
+        libpci3 libsensors5 \
+        python3.7 python3-pip \
```

Comment:
I would suggest don't use python3-pip.
Because it will download another python3 packages, which could conflict with the packages downloaded using pip3.

Reply:
Could you share more insight on what are "another python3 packages" and "the packages downloaded using pip3"?

---
In reply to: [419767478](https://github.com/Azure/sonic-buildimage/pull/4529#discussion_r419767478) [](ancestors = 419767478)

Reply:
I had an experience before. When you mixed up python packages, installed from the distribution and using pip you can get in the situation, when the distribution py package requires you to install one version of dependency, and pypi py package requires you to install another version of dependency. So usually I install pip and then install everything using pip, minimizing py packages from distribution.

Also in this case we already had working solution to download pip from the sources. Why do we need to change something which works?

Reply:
I feel it is better we use the packages provided by the distribution.
I haven't seen any issues due to using python3-pip

Reply:
Usually I try to keep changes as small as possible. I don't know what was the reasoning of the authors of the code, who downloaded pip from the Internet. I suggest to keep his approach

Reply:
It was me to download pip from Internet for python3.6 in a Debian Jessie environment. At that time it is difficult to build a python-pip 3.6. Since buster official support newer version of python3 and python3-pip, I think it is ok to install by `apt`.

---
In reply to: [420985982](https://github.com/Azure/sonic-buildimage/pull/4529#discussion_r420985982) [](ancestors = 420985982)